### ISSUE TITLE
Room - Settings - Save button is hidden when changing the room name/or tap fix the error message

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#14017]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Issue found when executing PR** https://github.com/Expensify/App/pull/13923

## Action Performed:
1. Launch the app
2. Log in with any expensifail account
3. Create a new room with name 123, select the same WS
4. Create another room with name abc (select the same WS)
5. Change the room name 123 to abc
6. Tap Save button
7. Or tap on fix the error message

## Expected Result:
Save button is not hidden when changing the room name/or tap fix the error message

## Actual Result:
Save button is hidden when changing the room name/or tap fix the error message

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari

**Version Number:** 1.2.48.2

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/210804980-ff52f6e1-2846-4a97-8963-68caf6065b17.MP4

https://user-images.githubusercontent.com/93399543/210804989-44640456-0fdf-4a4f-b109-8e7d242d7f8e.mp4

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by jasperhuangg** [comment]
I don't think this is a bug per se–the height of the screen is simply too small to fit the "Save" button when the input error messages are also displayed above it.

cc @shawnborton This seems like more of a design issue–what do you is the best course of action in this scenario? I personally think it's fine to just close this out–the user can still dismiss the keyboard to get to the "Save" button. Happy to work on a fix if you can think of one though.
